spec/controllers/reservations_controller_spec.rb

```diff
@@ -29,8 +30,45 @@
     end
   end
 
+  describe "DELETE #destroy" do
+    context "when user is logged in" do
+      before do
+        sign_in_user
+        delete_destroy(wishlist.id, item.id, reservation.id)
```

Comment:
Well, are you *pretty sure* those statements are true, or you've checked and you're sure they are? ;-)
But joking aside, I'm looking for a confirmation on this. If you find a confirmation for one way or the other, please provide as well.